error when making proofs with fcomm

I have occasionally observed transient errors when making proofs with fcomm. See example below. I also observe a signal 11. The errors I saw were not immediately reproducible.

```
➜  examples git:(improve-makefile) ✗ time make fibonacci-proof
cargo run --release -- prove --expression fibonacci.lurk --proof fibonacci-proof.json --lurk
    Finished release [optimized] target(s) in 0.13s
     Running `/Users/clwk/fil/lurk-rs/target/release/fcomm prove --expression fibonacci.lurk --proof fibonacci-proof.json --lurk`
libc++abi: terminating with uncaught exception of type std::__1::system_error: mutex lock failed: Invalid argument
make: *** [fibonacci-proof.json] Abort trap: 6
make fibonacci-proof  73.70s user 3.01s system 292% cpu 26.260 total
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -189,6 +189,5 @@ name = "public_params"
 harness = false
 
 [patch.crates-io]
-sppark = { git = "https://github.com/supranational/sppark", rev="5fea26f43cc5d12a77776c70815e7c722fd1f8a7" }
 # This is needed to ensure halo2curves, which imports pasta-curves, uses the *same* traits in bn256_grumpkin
 pasta_curves = { git="https://github.com/lurk-lab/pasta_curves", branch="dev" }

